backend/inspirehep/matcher/api.py

```diff
@@ -113,6 +115,9 @@ def match_reference_config(reference):
         config_data,
     ]
 
+    for config in configs:
+        config.update({"index": get_prefixed_index(config.get("index"))})
```

Comment:
maybe this was discussed and I'm missing something, but why not move this to the `inspire-matcher` library, to make this transparent to the caller? Note that it already depends on Flask due to using `invenio-search`, so it can read the config.

Reply:
Yes, seems good to me. What do you think @drjova?

Reply:
Totally agree, the change in the refinement it was mentioning the change to be in the `inspire-matcher`. We should close this PR and do it directly there.